data/data/ovirt/affinity_group/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "ovirt_affinity_group" "cluster_masters_affinity_group" {
+  count        = length(var.ovirt_control_plane_affinity_groups)
+  name         = var.ovirt_control_plane_affinity_groups[count.index]["name"]
+  cluster_id   = var.ovirt_cluster_id
+  priority     = var.ovirt_control_plane_affinity_groups[count.index]["priority"]
+  vm_positive  = false
+  vm_enforcing = false
```

Comment:
`vm_enforcing=false` means that the masters MAY end up on the same physical host if the user does not have enough hosts instead of failing. This can lead to inadvertently having a single point of failure.

This option should be customizable and I would strongly argue for setting the default value to `true` to stay in line with all Kubernetes deployment recommendations that I know of.

Reply:
Making this configurable and default to true